app/services/GraylogService.scala

```diff
@@ -0,0 +1,80 @@
"highlight_ranges\": {},\n    \"message\": {\n      \"process_id\": \"343\",\n      \"gl2_accounted_message_size\": 333,\n      \"level\": 3,\n      \"gl2_remote_ip\": \"10.0.10.172\",\n      \"gl2_remote_port\": 34370,\n      \"streams\": [\"000000000000000000000001\"],\n      \"gl2_message_id\": \"01EJ4QQ0VEF9VA8ZR3A9630ZCK\",\n      \"source\": \"7655c64956f5\",\n      \"message\": \"2020-09-13 22:00:07,277 [Thread-68      ] INFO    : pyclowder.connectors - [5f5e96668f08aa01dd47632b] : StatusMessage.done: Done processing\",\n      \"gl2_source_input\": \"5f2d6cbdfe6eb70769683d1c\",\n      \"application_name\": \"clowder-dev_ncsa_file_digest.1.irowe4759lbuoyl6r\",\n      \"gl2_source_node\": \"f1971377-bf1a-4d91-a30b-f784a03946bb\",\n      \"_id\": \"7c3f38e1-f60c-11ea-9775-0242ac120006\",\n      \"facility\": \"user-level\",\n      \"timestamp\": \"2020-09-13T22:00:07.000Z\"\n    },\n    \"index\": \"graylog_450\",\n    \"decoration_stats\": null\n  }, {\n    \"highlight_ranges\": {},\n    \"message\": {\n      \"process_id\": \"343\",\n      \"gl2_accounted_message_size\": 399,\n      \"level\": 3,\n      \"gl2_remote_ip\": \"10.0.10.172\",\n      \"gl2_remote_port\": 34370,\n      \"streams\": [\"000000000000000000000001\"],\n      \"gl2_message_id\": \"01EJ4QQ0JSZ8T3HBBSF3KVJ9GV\",\n      \"source\": \"7655c64956f5\",\n      \"message\": \"2020-09-13 22:00:06,998 [Thread-68      ] DEBUG   : __main__ - sending request for digest streaming: https://dev.simpl.syngentadigitalapps.com/clowder/api/files/5f5e96668f08aa01dd47632b/blob?key=FLzYnfJ6hl\",\n      \"gl2_source_input\": \"5f2d6cbdfe6eb70769683d1c\",\n      \"application_name\": \"clowder-dev_ncsa_file_digest.1.irowe4759lbuoyl6r\",\n      \"gl2_source_node\": \"f1971377-bf1a-4d91-a30b-f784a03946bb\",\n      \"_id\": \"7c14f490-f60c-11ea-9775-0242ac120006\",\n      \"facility\": \"user-level\",\n      \"timestamp\": \"2020-09-13T22:00:06.000Z\"\n    },\n    \"index\": \"graylog_450\",\n    \"decoration_stats\": null\n  }, {\n    \"highlight_ranges\": {},\n    \"message\": {\n      \"process_id\": \"5822\",\n      \"gl2_accounted_message_size\": 338,\n      \"level\": 3,\n      \"gl2_remote_ip\": \"10.0.10.241\",\n      \"gl2_remote_port\": 33366,\n      \"streams\": [\"000000000000000000000001\"],\n      \"gl2_message_id\": \"01EJ4GV93STR68E7FRCRM7Q1A5\",\n      \"source\": \"b6ae36cb3653\",\n      \"message\": \"2020-09-13 20:00:27,105 [Thread-68      ] INFO    : pyclowder.connectors - [5f5e7a458f08aa01dd476172] : StatusMessage.start: Started processing\",\n      \"gl2_source_input\": \"5f2d6cbdfe6eb70769683d1c\",\n      \"application_name\": \"clowder-dev_ncsa_file_digest.2.s6i35ypkuyzau2pr8\",\n      \"gl2_source_node\": \"f1971377-bf1a-4d91-a30b-f784a03946bb\",\n      \"_id\": \"b869f690-f5fb-11ea-9775-0242ac120006\",\n      \"facility\": \"user-level\",\n      \"timestamp\": \"2020-09-13T20:00:27.000Z\"\n    },\n    \"index\": \"graylog_450\",\n    \"decoration_stats\": null\n  }]\n}"
```

Comment:
@bingzhang what's this for?

Reply:
This is my debug code. I will make another pull request to remove them.